frontend/app/components/create-badge.js

```diff
@@ -4,19 +4,23 @@ const EmberObject  = Ember;
 
 export default Component.extend({
   didInsertElement() {
-    EmberObject.$(window).on('resize scroll', function(e) {
-      if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && this.scrollY < 1278) {
-        EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.position = 'fixed';
-        EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.right = '0';
-        EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.top = '0';
-      } else if (e.type === 'scroll' && this.window.innerWidth >= 764 && this.scrollY >= 1278) {
-        EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.position = 'fixed';
-        EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '570px';
-      } else {
-        EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px';
-        EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.position = 'inherit';
-
+    EmberObject.$(window).bind('resize scroll', function(e) {
+      if (EmberObject.$(document)[0].getElementsByClassName('six wide column preview').length != 0) {
+        if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && this.scrollY < 1278) {
+          EmberObject.$(document)[0].getElementsByClassName('six wide column preview')[0].style.position = 'fixed';
```

Comment:
Create a variable and then used at different places. Modularisation :sweat_smile: